docs/src/components/productJoyUI/JoyUIHero.tsx

```diff
@@ -0,0 +1,87 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import HeroContainer from 'docs/src/layouts/HeroContainer';
+import IconImage from 'docs/src/components/icon/IconImage';
+import GetStartedButtons from 'docs/src/components/home/GetStartedButtons';
+import GradientText from 'docs/src/components/typography/GradientText';
+import Link from 'docs/src/modules/components/Link';
+import ROUTES from 'docs/src/route';
+import dynamic from 'next/dynamic';
+
+const DesktopShowcase = dynamic(() => import('./HeroShowcase').then((mod) => mod.Desktop), {
+  ssr: false,
+});
+const MobileShowcase = dynamic(() => import('./HeroShowcase').then((mod) => mod.Mobile), {
+  ssr: false,
+});
+
+export default function JoyUIHero() {
+  return (
+    <HeroContainer
+      linearGradient
+      left={
+        <Box sx={{ textAlign: { xs: 'center', md: 'left' }, ml: { xl: '-40px' } }}>
+          <Typography
+            fontWeight="bold"
+            variant="body2"
+            sx={(theme) => ({
+              color: 'primary.600',
+              display: 'flex',
+              alignItems: 'center',
+              gap: 1,
+              justifyContent: { xs: 'center', md: 'flex-start' },
+              ...theme.applyDarkStyles({
+                color: 'primary.300',
+              }),
+            })}
+          >
+            <IconImage width={28} height={28} name="product-core" />{' '}
+            <Link href={ROUTES.productCore}>MUI Core</Link>{' '}
+            <Typography component="span" variant="inherit" sx={{ color: 'divider' }}>
+              /
+            </Typography>
+            <Typography component="span" variant="inherit" sx={{ color: 'text.primary' }}>
+              Joy UI
+            </Typography>
+          </Typography>
+          <Typography
+            variant="h1"
+            sx={{
+              my: 2,
+              maxWidth: { xs: 500, md: 'unset' },
+              minWidth: { lg: 650 },
+              position: 'relative',
+              zIndex: 1,
+            }}
+          >
+            Craft gorgeous UIs
+            <br /> that <GradientText>spark joy</GradientText>
+          </Typography>
+          <Typography color="text.secondary" sx={{ mb: 3, maxWidth: 500 }}>
+            Joy UI is for those that appreciate the comprehensiveness and reliability of Material
+            UI, but don&apos;t want Material Design. It&apos;s design agnostic and built to be
+            tailored to your specific design language.
```

Comment:
I think we should avoid all mention of Material UI and Material Design here in the hero section. That can come later. We should use the opportunity to describe what it _is_ more than what it's _not_. What are the top two or three key value propositions worth highlighting here? "Joy UI is great because..."

Reply:
Agreed, yeah! Meta note: Every piece of copy we're using on the page at the moment is either what we iterated on while the page was still in the Figma stage or place-holder stuff I came up with — so, taking advantage of this comment to extend the invite to revise the entire page's copywriting! 🤙 Some comments about specific parts were already dropped, and I'd like to get your take on them (click on the "Load more" up there a few times to see the previous thread 😬).

For this comment specifically, though, I'd likely highlight these aspects: 1) UI design (minimal, sleek, lightly opinionated); 2) customization UX (CSS variables, automatic features, etc.); 3) either DX (which can be covered in the previous topic) or breadth of components. Not fully sure about the third...